### PR TITLE
Add jupyter_paperboy recipe

### PR DESCRIPTION
**Pull request by blink1073**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by blink1073** [comment]
cc @timkpaine, please acknowledge that you'd like to be a maintainer.

**Comment by timkpaine** [comment]
ACK

**Comment by blink1073** [comment]
Drat, a few of the package dependencies aren't on conda-forge:

```
conda.exceptions.ResolvePackageNotFound: 
  - falcon-cors[version='>=1.1.7']
  - falcon-helpers[version='>=0.17.0']
  - falcon-multipart[version='>=0.2.0']
```

**Comment by blink1073** [comment]
The good news is that `falcon-helpers` depends on `falcon-multipart`, but otherwise all of their deps are already on conda-forge, so it is just those three.

**Comment by blink1073** [comment]
It looks like conda-forge requires the license file to be included upstream.  I opened the following:

https://github.com/yohanboniface/falcon-multipart/pull/15
https://github.com/lwcolton/falcon-cors/pull/15
https://gitlab.com/skosh/falcon-helpers/merge_requests/18/

**Comment by blink1073** [comment]
@jakirkham, is it a strict requirement to have the license file included?  We're trying to add 3 libraries.  2 of them have not had a release in 2+ years, and 2 of my PRs are failing due to unrelated reasons.

**Comment by blink1073** [comment]
cf https://github.com/conda-forge/staged-recipes/pull/9014

**Comment by timkpaine** [comment]
@blink1073 please poke this :-)

**Comment by blink1073** [comment]
@conda-forge/help-python this is ready for review.

**Comment by blink1073** [comment]
Thanks @mariusvniekerk! :heart:

File: recipes/jupyter_paperboy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - falcon >=1.4.1
+    - gunicorn >=19.9.0  # [not win]
+    - waitress >= 1.2.1  # [win]
```
Comment (isuruf) [inline recipes/jupyter_paperboy/meta.yaml]: There should be no space before the version.

**Pull request merged**